src/SSHDebugPS/AD7/AD7Port.cs

```diff
@@ -65,24 +66,26 @@ public bool IsConnected
 
         public int EnumProcesses(out IEnumDebugProcesses2 processEnum)
         {
+            int hr = HR.S_OK;
             IEnumDebugProcesses2 result = null;
             var connection = GetConnection();
 
             if (connection == null)
             {
-                processEnum = null;
-                return HR.E_REMOTE_CONNECT_USER_CANCELED;
+                // Don't return a failure to prevent vsdebug.dll from showing an error message
```

Comment:
@gregg-miskelly I verified that this only gives me one error message. `null` doesn't work as it generates a different error.